.travis.yml

```diff
@@ -15,6 +11,14 @@ env:
     - ROS_BUILDFARM_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
 matrix:
   include:
+    - language: python
+      python: "3.6"
+      sudo: false
+      install:
+        - pip install catkin-pkg EmPy PyYAML rosdistro vcstool
+        - pip install coverage "flake8<3.7" flake8-docstrings flake8-import-order nose pep8 pyflakes
```

Comment:
Why `flake8 < 3.7`? Also `coverage` doesn't seem to be used.

It would be good to install more `flake8` plugins (like the ones we use in `ament_flake8`). But that can be a separate PR.

Reply:
`flake8` 3.7 includes at least one breaking change to the legacy API, and my machine didn't have 3.7, and the original tests didn't target it. So it was the least amount of effort.

I went ahead and made the corresponding change to `test_flake8.py`.

I can't see any flake8 plugin references in `ament_flake8`. If you want those included in this PR, you'll have to be more specific, but this is already a pretty big batch of goodness without them.

Reply:
Most of the plugins are not available as Debian packages therefore they are not listed in the package manifest. I was referring to the plugins we install in CI: https://github.com/ros2/ci/blob/3eb0842d60d4a88aaaa02a0acbf02485c4f364aa/ros2_batch_job/__main__.py#L53-L61

But as said above: this can be done in a separate PR.

Reply:
> this can be done in a separate PR

Cool, sounds good.